client/gutenboarding/controller.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import config from 'config';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { Gutenboard } from './gutenboard';
+
+export function redirectIfNotEnabled( context, next ) {
+	if ( ! config.isEnabled( 'gutenboarding' ) ) {
+		page.redirect( '/' );
```

Comment:
Needs a `return` here right? 

```suggestion
		page.redirect( '/' );
		return;
```